.github/workflows/_deploy-shared.yml

```diff
@@ -12,6 +12,9 @@ on:
       app_name_admin:
         required: true
         type: string
+      app_name_dashboard:
+        required: false
+        type: string

```

Comment:
I reckon we should have a dashboard for each environment - so we can test changes to the dashboard in sandbox before we roll them out to production.
If we do that, we could make this field required (same as the other `grc` and `admin` app names)